use-cases/bed-allocation/src/main/java/org/acme/bedallocation/rest/DemoDataGenerator.java

```diff
@@ -52,7 +52,7 @@ public class DemoDataGenerator {
     private final Random random = new Random(0);
 
     /**
-     * The dataset was generated based on the probability distributions found in the test dataset file overconstrained01.txt.
+     * The dataset was generated based on the probability distributions found in the test dataset file.
      * However, the number of patients was reduced to 40 to simplify the quickstart process.
      */
```

Comment:
Maybe this entire comment doesn't apply anymore?
It references a file nobody will be able to find.